CMake/CXX11.cmake

```diff
@@ -0,0 +1,48 @@
+# This is cloned from
+# https://github.com/nitroshare/CXX11-CMake-Macros
+# until C++11 support finally hits CMake stable (should be 3.1, I think).
+
+# Copyright (c) 2013 Nathan Osman
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# Determines whether or not the compiler supports C++11
+macro(check_for_cxx11_compiler _VAR)
```

Comment:
Maybe I'm mistaken, but I don't think cmake finds these macros unless directed to them. It doesn't look like you're using them quite yet anyway.

Reply:
So should I remove this file then ?

Reply:
Maybe? What were you hoping for it to do?

Reply:
it worked, thanks :)